RadixWallet/Features/SettingsFeature/Preferences/Preferences+View.swift

```diff
@@ -0,0 +1,182 @@
+private typealias S = L10n.Preferences
+
+extension Preferences.State {
+	var viewState: Preferences.ViewState {
+		let isDeveloperModeEnabled = appPreferences?.security.isDeveloperModeEnabled ?? false
+		#if DEBUG
+		return .init(
+			isDeveloperModeEnabled: isDeveloperModeEnabled,
+			exportLogsUrl: exportLogsUrl
+		)
+		#else
+		return .init(
+			isDeveloperModeEnabled: isDeveloperModeEnabled
+		)
+		#endif
+	}
+}
+
+// MARK: - Preferences.View
+
+public extension Preferences {
+	struct ViewState: Equatable {
+		let isDeveloperModeEnabled: Bool
+		#if DEBUG
+		let exportLogsUrl: URL?
+		init(isDeveloperModeEnabled: Bool, exportLogsUrl: URL?) {
+			self.isDeveloperModeEnabled = isDeveloperModeEnabled
+			self.exportLogsUrl = exportLogsUrl
+		}
+		#else
+		init(isDeveloperModeEnabled: Bool) {
+			self.isDeveloperModeEnabled = isDeveloperModeEnabled
+		}
+		#endif
+	}
+
+	@MainActor
+	struct View: SwiftUI.View {
+		private let store: StoreOf<Preferences>
+
+		public init(store: StoreOf<Preferences>) {
+			self.store = store
+		}
+
+		public var body: some SwiftUI.View {
+			content
+				.setUpNavigationBar(title: S.title)
+				.tint(.app.gray1)
+				.foregroundColor(.app.gray1)
+				.presentsLoadingViewOverlay()
+				.destinations(with: store)
+		}
+	}
+}
+
+extension Preferences.View {
+	@MainActor
+	private var content: some View {
+		WithViewStore(store, observe: \.viewState, send: { .view($0) }) { viewStore in
+			ScrollView {
+				VStack(spacing: .zero) {
+					ForEach(rows(viewStore: viewStore)) { kind in
+						kind.build(viewStore: viewStore)
```

Comment:
I don't think that a model should have a func that returns a view, I would greatly prefer having say a RowView that takes a kind, and have the switch in there.